### PR TITLE
DO-437-UX fixes(17 July)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/member/MessageBox/MessageBox.jsx [hunk 1]
```diff
@@ -87,14 +89,62 @@ const MessageBox = ({ onClickSent, onCloseMessageBox, showCloseButton, occasionT
...
   return (
     <Container className="message-box-container">
       {renderMessageBoxCloseButton()}
       <ResizableTextarea value={message} onChangeMessage={onChangeMessageText} messageText={messageText} />
-      <Container className="message-box-container__button-container">
-        <Button value="Send" onClick={() => { onClickSent(message); }} />
-      </Container>
-    </Container>);
+      <SendButton />
+      {/*<Container className="message-box-container__button-container">*/}
```
Comment (mohitsaladiya) [inline src/components/member/MessageBox/MessageBox.jsx:133]: Commented out code??

File: src/components/member/MessageBox/MessageBox.jsx [hunk 2]
```diff
@@ -9,6 +9,8 @@ import IconCross from '../../common/Typography/Icons/IconCross';
 import { OCCASION_TYPE } from '../../../constants/common/commonConstants';
```
Comment (mohitsaladiya) [inline src/components/member/MessageBox/MessageBox.jsx:9]: Why message box changes are committed in this PR?

File: src/components/member/MessageBox/MessageBox.jsx (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/member/MessageBox/MessageBox.jsx:133]: Done

File: src/components/member/MessageBox/MessageBox.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/member/MessageBox/MessageBox.jsx:9]: It is used to apply condition in wish button to display cake icon on birthdays and ring icon on anniversaries.

**Pull request merged**